tests/corpus/parsing/test_cha_parser.py

```diff
@@ -0,0 +1,95 @@
+import os
+import tempfile
+import pytest
+import requests
+import sktalk
+from sktalk.corpus.parsing.cha import ChaParser
+from sktalk.corpus.parsing.parser import Parser
+
+
+class TestParser:
+    milliseconds_timestamp = [
+        ["0", "00:00:00.000"],
+        ["1706326", "00:28:26.326"],
+        ["222222", "00:03:42.222"],
+        ["None", None]
+    ]
+
+    @pytest.mark.parametrize("milliseconds, timestamp", milliseconds_timestamp)
+    def test_to_timestamp(self, milliseconds, timestamp):
+        assert Parser._to_timestamp(milliseconds) == timestamp
```

Comment:
As indicated above, migrating `Parser._to_timestamp()` to `timeutils.to_timestamp()` would resolve the issue of testing private (static) methods here.